plio/io/io_gdal.py

```diff
@@ -146,8 +152,6 @@ def __init__(self, file_name):
 
         """
         self.file_name = file_name
-        if not gdal:
-            raise ImportError('No module name gdal.')
```

Comment:
Why are we not erroring here anymore?

Reply:
Ah yes, I see that was an ImportError before my changes in 58d36f8.

This was my way of trying to bandaid over the Travis-CI issue I was having here:
https://travis-ci.org/github/USGS-Astrogeology/plio/jobs/765206817

Basically, I'm trying to get plio to look at either plio.io for goal, then if that doesn't work, look in osgeo. For some reason, the remote Travis-CI builds of libpyhat cannot find gdal in plio.io.

Reply:
I would leave this file unchanged if at all possible and modify the gdal import stuff in the init file:

https://github.com/USGS-Astrogeology/plio/blob/master/plio/io/__init__.py#L6

Reply:
We also want this error to be here because we should not be using the geodataset without gdal.

Reply:
I tried doing this in __init__.py locally, but it wouldn't pass tests. I'll take a stab at it and send the PR to see how it fares in the remote build. Reverting those changes to io_gdal as well. Stand by..